src/hermes/model/path.py

```diff
@@ -0,0 +1,372 @@
+import logging
+import typing as t
+
+import pyparsing as pp
+
+from hermes.model import errors
+
+_log = logging.getLogger('hermes.model.path')
+
+
+class ContextPathGrammar:
+    """
+    The pyparsing grammar for ContextGrammar paths.
+    """
+
+    key = pp.Word('@' + pp.alphas)
+    index = pp.Word(pp.nums).set_parse_action(lambda tok: [int(tok[0])]) | pp.Char('*')
+    field = key + (pp.Suppress('[') + index + pp.Suppress(']'))[...]
+    path = field + (pp.Suppress('.') + field)[...]
+
+    @classmethod
+    def parse(cls, text: str) -> pp.ParseResults:
+        """
+        Parse a ContextPath string representation into its individual tokens.
+
+        :param text: The path to parse.
+        :return: The pyparsing.ParseResult.
+        """
+        return cls.path.parse_string(text)
+
+
+class ContextPath:
+    """
+    This class is used to access the different contexts.
+
+    On the one hand, the class allows you to define and manage pathes.
```

Comment:
```suggestion
    On the one hand, the class allows you to define and manage paths.
```

**typo:** :tada: for fixing 💋